_topics/en/swap-in-potentiam.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Swap-in Potentiam (SIP)
+
+## Optional. test Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: swap-in potentiam
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+topic-categories:
+  - Contract Protocols
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Swap-in-Potentiam (SIP)** is a protocol that facilitates the immediate transfer of confirmed
+  on-chain Bitcoin to the Lightning Network. It reduces trust requirements compared to other
+  instant channel opening methods like 0-conf channels by temporarily committing to co-ownership with a Lightning Service
+  Provider (LSP) and delaying unilateral access per a timeout.
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+  - title: Moving Onchain Funds "Instantly" To Lightning (Jesse Posner, ZmnSCPxj)
+    link: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-January/003810.html
+
+  - title: LSPS C= Extension, Swap-in-Potentiam
```

Comment:
@arshbot in going through the content one last time before merge, I saw this is a dead link and it doesn't appear there is an equivalent repo under https://github.com/ZmnSCPxj-jr/

Is there another different link you are aware of? Or should we remove this reference?

Reply:
Dropped this link for now.